src/connectors/tunegenie.ts

```diff
@@ -0,0 +1,8 @@
+export {};
+
+Connector.playerSelector = '#back_matte';
+Connector.trackSelector = 'ul.currentonair li.slot:first-child .song';
+Connector.artistSelector =
+	'.currentonair > li.slot:first-child .left div:not(.song)';
+Connector.trackArtSelector = '.currentonair > li.slot:first-child .disc img';
+Connector.isPlaying = () => Util.hasElementClass('.play.btn i', 'icon-stop');
```

Comment:
```suggestion
Connector.playButtonSelector = '.play.btn .icon-play';
```
I don't think the original here will consistently work between live and playing back playlists. I think this should do the trick, but is there some issue I didn't spot? Disclaimer I have not run this.

Reply:
@yayuyokitano I changed both is playing implementations. The tunegenie.ts to your suggestion, and made the second one that relies on the SVG to check if it's not the play button code instead of if it is the stop button. This allowed it to handle the podcast playback. Unfortunately the only thing that changes on play/pause is the SVG code and a couple inline styles, so not really any more reliable. It can pickup the podcasts, but I won't say it's great as they're usually 60 minute shows that aren't broken up into individual songs. It can also pickup the 30 second song previews, but it won't scrobble them due to the length.

Reply:
Cool, will merge this then, the player finickyness can always be worked around better at a later time if someone figures something out